### PR TITLE
pkg: make JavaScript keyspacing consistent

### PR DESCRIPTION
**Pull request by jelly**
cockpit-ostree enables this by default and we have to disable it now. Having consistent key spacing for all our JavaScript objects seems like something we want everywhere as eslint --fix can automatically.

After this landed, we should also apply this to machines/podman and again to ostree.

### CODE REVIEW

**Comment by martinpitt** [comment]
This is good, but please don't block on it. eslint really shouldn't test pkg/lib/ in our extension projects.

File: pkg/storaged/dialog.jsx [hunk 1]
```diff
@@ -1022,9 +1022,9 @@ export const SizeSlider = (tag, title, options) => {
 
 export const BlockingMessage = (usage) => {
     const usage_desc = {
-        pvol:                  _("physical volume of LVM2 volume group"),
-        mdraid:                _("member of RAID device"),
-        vdo:                   _("backing device for VDO device"),
+        pvol: _("physical volume of LVM2 volume group"),
+        mdraid: _("member of RAID device"),
+        vdo: _("backing device for VDO device"),
```
Comment (cockpituous) [inline pkg/storaged/dialog.jsx:1027]: These 3 added lines are not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-17876-20221104-114502-d7b5a827-fedora-36-devel//Coverage/lcov/github-pr.diff.gcov.html)

File: pkg/storaged/dialog.jsx [hunk 2]
```diff
@@ -1048,7 +1048,7 @@ export const BlockingMessage = (usage) => {
                           columns={[
                               { title: _("Device") },
                               { title: _("Location") },
-                              { title:  _("Use") }
+                              { title: _("Use") }
```
Comment (cockpituous) [inline pkg/storaged/dialog.jsx:1051]: This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-17876-20221104-114502-d7b5a827-fedora-36-devel//Coverage/lcov/github-pr.diff.gcov.html)

File: pkg/users/account-create-dialog.js [hunk 3]
```diff
@@ -119,22 +119,22 @@ function validate_username(username, accounts) {
...
+            l: '[ĺľ]',
+            n: '[ňñ]',
+            o: '[òóôõö]',
+            oe: 'œ',
+            r: '[ŕř]',
+            s: 'š',
+            t: 'ť',
+            u: '[ùúůûűü]',
+            y: '[ýÿ]',
+            z: 'ž',
```
Comment (cockpituous) [inline pkg/users/account-create-dialog.js:137]: These 16 added lines are not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-17876-20221104-114502-d7b5a827-fedora-36-devel//Coverage/lcov/github-pr.diff.gcov.html)

**Comment by jelly** [comment]
> This is good, but please don't block on it. eslint really shouldn't test pkg/lib/ in our extension projects.

Oh, no plans to block on it. This was more a quick PR to see how easy it was :) (It is easy if you don't forget to eslint --fix pkg/**/*.{js,jsx}

**Comment by jelly** [comment]
Rebased to fix the conflict.

File: pkg/storaged/jobs-panel.jsx [hunk 4]
```diff
@@ -39,50 +39,50 @@ const _ = cockpit.gettext;
  */
 
 const descriptions = {
-    'ata-smart-selftest':          _("SMART self-test of $target"),
-    'drive-eject':                 _("Ejecting $target"),
```
Comment (martinpitt) [inline pkg/storaged/jobs-panel.jsx:43]: These tabular views were actually kinda nice, but I suppose we'll have to bow to the machine linter bots.. :robot:

**Review by martinpitt — APPROVED** [review]

File: pkg/storaged/dialog.jsx (hunk 1 above)
Comment (cockpituous) [inline pkg/storaged/dialog.jsx:1027]: These 3 added lines are not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-17876-20221109-102136-9fe18085-fedora-36-devel//Coverage/lcov/github-pr.diff.gcov.html)

File: pkg/storaged/dialog.jsx (hunk 2 above)
Comment (cockpituous) [inline pkg/storaged/dialog.jsx:1051]: This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-17876-20221109-102136-9fe18085-fedora-36-devel//Coverage/lcov/github-pr.diff.gcov.html)

File: pkg/users/account-create-dialog.js (hunk 3 above)
Comment (cockpituous) [inline pkg/users/account-create-dialog.js:137]: These 16 added lines are not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-17876-20221109-102136-9fe18085-fedora-36-devel//Coverage/lcov/github-pr.diff.gcov.html)

**Pull request merged**